src/NServiceBus.Core/Transports/Msmq/MsmqDequeueStrategy.cs

```diff
@@ -305,31 +309,32 @@ void Action()
             }
         }
 
-        void LogCorruptedMessage(Message message, Exception ex)
+        static void LogCorruptedMessage(string messageId, string queueName, Exception ex)
         {
-            var error = string.Format("Message '{0}' is corrupt and will be moved to '{1}'", message.Id, ErrorQueue.Queue);
+            var error = $"Message '{messageId}' is corrupt and will be moved to '{queueName}'";
```

Comment:
Previously this was using `ErrorQueue.Queue` now I pass in `errorQueue.QueueName` which I think should be fine